src/components/App.js

```diff
@@ -2038,6 +2060,8 @@ class App extends Component {
             return
         }
 
+        try {logger.loadOnce()} catch (err) {}
```

Comment:
This seems like an arbitrary place to load something once. Can we avoid calling `logger.loadOnce()` altogether? Maybe we can call `loadOnce` automatically (in `logger` itself) when we call `writeToLogFileGTP()`?

Reply:
The placement has entirely to do with the desired behaviour of to give the user a warning/error of an invalid log folder path when an engine is attached.  I think it would be awkward if not bad if that warning fired when we actually tried writing to the log.  If you want to get rid of this warning message all together, however, we could move the loadOnce() into writeToLogFileGTP, by doing the load inside an `if (logFileNameGTP==null)`

Reply:
I see, then I prefer it to have it here.

Reply:
Ok, then there is nothing else I can do then.